src/app/(post-login)/submission/program/[shortName]/clinical-submission/components/FilesNavigator/StatsArea.tsx

```diff
@@ -8,7 +8,7 @@ import { ComponentProps } from "react";
 
 const StarIcon = DataTableStarIcon;
 
-type RecordState = "NEW" | "NONE" | "UPDATED" | "ERROR" | "WARNING";
+export type RecordState = "NEW" | "NONE" | "UPDATED" | "ERROR" | "WARNING";
```

Comment:
What's the logic to this ordering? not alphabetical or logical as far as I can tell. Maybe sort it alphabetically instead?